docs/openapi-docs/src/main/scala/sttp/tapir/docs/openapi/EndpointToOperationResponse.scala

```diff
@@ -104,4 +106,11 @@ private[openapi] class EndpointToOperationResponse(
       .defaultDecodeFailureOutput(input)
       .map(output => outputToResponses(output, ResponsesDefaultKey, None))
       .getOrElse(ListMap())
+
+  private def extractForcedContentType(outputs: List[EndpointOutput[_]]): Option[String] = outputs match {
```

Comment:
again, I'd call this `extractFixedContentType`

Reply:
another thing - outputs might be nested, e.g. using mapped pairs or simply pairs. The above will probably break given an output like `.out(header("C-T", "...").and(header("another", "...")))`

For this purposes, there's a `traverseOutputs` method which properly checks the entire structure